src/home-page/components/DemoChart.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+
+export const DemoChart: React.FC<{}> = () => (
```

Comment:
It looks lovely 😍
![image](https://user-images.githubusercontent.com/8350925/111353234-62207a00-8685-11eb-8c55-682ff759bc3e.png)